schemas/json/block.json

```diff
@@ -798,6 +798,20 @@
 				}
 			]
 		},
+		"viewScriptModule": {
+			"description": "Block type frontend script module definition. It will be enqueued only when viewing the content on the front of the site.",
+			"oneOf": [
+				{
+					"type": "string"
+				},
+				{
+					"type": "array",
+					"items": {
+						"type": "string"
+					}
```

Comment:
Thanks for the PR!
I need to test if we are supporting arrays at this moment.

Reply:
Thanks for the review! From what I've researched, it appears to support arrays.

- Document: https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-metadata.md#view-script-module
- PHPUnit test: https://github.com/WordPress/wordpress-develop/blob/7b24083859c37d8ac75b9549f458bc389610eaef/tests/phpunit/data/blocks/notice/block.json#L71